spec/site/tlc/io_spec.rb

```diff
@@ -21,11 +21,51 @@
       # 3. Wait for status = true  
       specify 'forcing is set with M0019', sxl: '>=1.0.13' do |example|
         Validator::Site.connected do |task,supervisor,site|
-          status = 'False'
-          input = 1
-          inputValue = 'True'
+
           prepare task, site
+
+          input = 101
```

Comment:
Consider reading input from config file `Validator.config['items']['inputs']` instead of using a hard coded value

For inspiration, check out the test case "is activated with M0006":

```ruby
inputs = Validator.config['items']['inputs']
skip("No inputs configured") if inputs.nil? || inputs.empty?
Validator::Site.connected do |task,supervisor,site|
    prepare task, site
    inputs.each { |input| switch_input input }
end
```